### DISCUSSION TITLE
Publishing a Paper in the EDC Publication

### DISCUSSION THREAD

**Discussion by git-masoud** [eclipse-edc/Publications#21] [Q&A]
Dear EDC community,

I would like to know the publication process for papers related to EDC. My colleagues and I have written a paper targeting users with backgrounds in IDS-Connector, which aims to demonstrate how EDC can be used as a replacement for IDS-Connector. The paper has been reviewed by Markus Spiekermann, who provided us with valuable feedback that we have incorporated into the final version of our paper. We would now like to explore the possibility of publishing it in the EDC publication.

We would appreciate any information you can provide regarding the submission process, including the required format and any guidelines or requirements we should be aware of. Additionally, we would welcome the opportunity to engage in a discussion with the EDC community about the content of our paper and the potential benefits of using EDC.

Thank you for your time and consideration. We look forward to hearing back from you.
Masoud

**Comment by PeterKoen-MSFT** [comment]
Hi Masoud,

Please, create a pull request and we will be happy to review it and publish if it fits into the EDC publications.

Thanks,
Peter
